README.md

```diff
@@ -0,0 +1,64 @@
+Power Board v4 Firmware
+=======================
+
+The Power Board distributes power to the SR kit from the battery. It
+provides six individual general-purpose power outputs along with two low
+current 5V power outputs.
+
+It also holds the internal On\|Off switch for the whole robot as well as
+the Start button which is used to start your robot code running.
+
+USB Interface
```

Comment:
I don't think this section is useful, and would probably be better placed in the docs or alike.

Reply:
It's mostly here as a summary. This isn't intended for competitor use. I think it's useful to explain the purpose of the board.

Reply:
I'm a big fan of storing the docs for code as close to the code as possible, so +1 to having the protocol documented in this repo.

Reply:
In hindsight, I kinda agree :+1: